packages/fbtee/src/formatNumber.tsx

```diff
@@ -1,6 +1,11 @@
 import { ReactElement } from 'react';
+import fbs from './fbs.tsx';
 import intlNumUtils from './intlNumUtils.tsx';
 
+// Ensure the local version of `fbs` is used instead of auto-importing `fbtee`.
+// eslint-disable-next-line no-unused-expressions, @typescript-eslint/no-unused-expressions
+fbs;
+
```

Comment:
This was necessary because this module uses `<fbs>`, which was also not compiled previously. However, it is currently not actually part of the distributed files anyway.

(We may remove this module depending on what we decide to do with Number formatting)

Reply:
We now have an auto import plugin which will import `fbtee`, which always resolves to `lib/index.js` (the build output). That means we will end up with two copies of the library in tests since it will load the source and then auto-import the build, therefore breaking everything. This ensures that the local version is imported, and the import is not stripped since the binding is "used".